packages/react-mutation-mapper/src/component/track/TrackPanel.tsx

```diff
@@ -102,4 +98,42 @@ export default class TrackPanel extends React.Component<TrackPanelProps, {}> {
             </div>
         );
     }
+
+    private getPtmTrack(trackName: TrackName, ptmSource: PtmSource) {
+        return !this.props.trackVisibility ||
+            this.props.trackVisibility[trackName] === 'visible' ? (
+            <PtmTrack
+                store={this.props.store}
+                pubMedCache={this.props.pubMedCache}
+                ensemblTranscriptId={
+                    this.props.store.activeTranscript &&
+                    this.props.store.activeTranscript.result
+                        ? this.props.store.activeTranscript.result
+                        : undefined
+                }
```

Comment:
Maybe let's shorten it to:
```
ensemblTranscriptId={
                    this.props.store.activeTranscript?.result
                }
```

Reply:
yeah, is there really a case where activeTranscript would be undefined itself?

Reply:
Theoretically yes. `activeTranscript` is defined as an optional field for `MutationMapperStore` interface: https://github.com/cBioPortal/cbioportal-frontend/blob/master/packages/react-mutation-mapper/src/model/MutationMapperStore.ts#L30

Can't remember if we have an actual store without an `activeTranscript` implementation though.